build.gradle

```diff
@@ -33,17 +33,20 @@ dependencies {
 
     api ('org.seleniumhq.selenium:selenium-api') {
         version {
-            strictly "${seleniumVersion}"
+            strictly "[${seleniumVersion}, 5.0)"
+            prefer "${seleniumVersion}"
         }
     }
     api ('org.seleniumhq.selenium:selenium-remote-driver') {
         version {
-            strictly "${seleniumVersion}"
+            strictly "[${seleniumVersion}, 5.0)"
```

Comment:
not sure if we have to put 4.0 there instead...
`seleniumVersion` in such case would be the version we tested with and verified working, so it might be higher than 4.0, e.g. 4.1, although it does not mean that 4.0 should necessarily stop working